client/src/components/new-request/search.vue

```diff
@@ -457,6 +449,9 @@ export default class NewSearch extends Vue {
 #goto-corporate-btn {
   min-height: 45px !important;
 }
+#name-input-component {
+  margin-top: 0 !important;
```

Comment:
Since this element only has this one style, you might be able to get rid of this and apply the class of `mt-0` on the element itself. I think* it does the same as `!important`

Either way, super small :)

Reply:
I did try that, but for some reason it gets overwritten by a different !important with mt=-20 after the errors are added. I didn't figure out why that was happening, but this gave the mt=0 more priority